app/models/concerns/file_set/geo_file_format_behavior.rb

```diff
@@ -38,20 +30,23 @@ def vector_work?
   end
 
   module ClassMethods
-    def image_file_formats
-      ['TIFF', 'IMAGE_FILE']
```

Comment:
Thiking ahead to #87, do we want to keep a controlled list of image file format supported?